pytoda/preprocessing/tests/test_crawlers.py

```diff
@@ -73,6 +77,27 @@ def test_get_smiles_from_pubchem(self) -> None:
                 )
                 self.assertEqual(smiles, ground_truth)
 
+    def test_query_pubchem(self) -> None:
+        """Test query_pubchem"""
+        smiles_list = [
+            'O1C=CC=NC(=O)C1=O', 'CC(N)S(O)(=O)C(C)CC(C(C)C)c1cc(F)cc(F)c1',
+            'Clc1ccccc2ccnc12'
+        ]
+        ground_truths = [(True, 67945516), (False, -1), (False, -1)]
+        for gt, smiles in zip(ground_truths, smiles_list):
+            self.assertTupleEqual(query_pubchem(smiles, return_id=True), gt)
```

Comment:
```suggestion
            self.assertTupleEqual(query_pubchem(smiles), gt)
```
according to the suggested changes in signature of `query_pubchem`.